src/utils/kernel_name_shortener.py

```diff
@@ -23,18 +23,18 @@
 ##############################################################################el
 
 import os
-import glob
+import logging
 import re
 import subprocess
 import pandas as pd
 
-from utils.utils import console_error, console_debug, console_log
+from utils.utils import console_error, console_log
```

Comment:
Keep `console_debug()`
```suggestion
from utils.utils import console_error, console_log, console_debug
```